cmd/enr.go

```diff
@@ -50,8 +50,9 @@ func newEnrCmd(runFunc func(io.Writer, p2p.Config, string) error) *cobra.Command
 // runNewENR loads the p2pkey from disk and prints the ENR for the provided config.
 func runNewENR(w io.Writer, config p2p.Config, dataDir string) error {
 	key, err := p2p.LoadPrivKey(dataDir)
+	helpMsg := fmt.Sprintf("No ENR private key found in %s. If this is your first time running this client, create one with `charon create enr`.", p2p.KeyPath(dataDir))
 	if err != nil {
-		return err
+		return errors.New(helpMsg)
```

Comment:
You are assuming that this is the only error that can occur, which is not true. Also please use structured errors always.

Suggest
```
key, err := p2p.LoadP2PKey(dir)
if os.IsNotExist(err) {
  return errors.New("ENR private key not found. Maybe create one with `charon create enr`", z.Str("path", p2p.KeyPath(dataDir))
} else if err != nil {
  return err
} 
return errors.Wrap(err, ""
```
Also please add a test for this!

Reply:
For `os.IsNotExist(err)` to work, the error coming from calling `p2p.LoadPrivKey` should not be wrapped. wrapping changes the error type from `*fs.PathError` to `structured`.

But, when i attempt to simply return err, like, 
```go

func LoadPrivKey(dataDir string) (*ecdsa.PrivateKey, error) {
	key, err := crypto.LoadECDSA(KeyPath(dataDir))
	if err != nil {
		// return nil, errors.Wrap(err, "load key")
                // But, this throws an error by golangci-lint's wrapcheck linter
                	return nil, err
	}

	return key, nil
}
```